### PR TITLE
Add tooltips to select and multi-select

### PR DESCRIPTION
**Pull request by philpee2**
By default, options in the select and multi-select have tooltips. This can be
controlled with the `showTooltip` property.

Also replaces `yes` and `no` in documentation with `true` and `false`

@Addepar/uie @igillis @thangdinh

### CODE REVIEW

**Comment by thangdinh** [comment]
C/c: @azirbel

**Comment by philpee2** [comment]
@thangdinh Alex should be getting tagged in the UIE group, no?

File: app/templates/ember_widgets/documentation.hbs [hunk 1]
```diff
@@ -100,6 +100,11 @@
       <td>&#39;&#39;</td>
       <td><p>The path of the default option.</p></td>
     </tr>
+    <tr>
+      <td>showTooltip</td>
+      <td>true</td>
+      <td><p>Whether to show a tooltip when hovering over an option.</p></td>
```
Comment (hanakimis) [inline app/templates/ember_widgets/documentation.hbs]: I would check with Patricia on this wording. Don't think she has access to ember widgets, but in the meantime, might make sense to just pull her aside for advice...

Comment (Dylnuge) [inline app/templates/ember_widgets/documentation.hbs]: @pagael

It's open source so she should be able to see this even if she can't write to the repo.

Comment (pagael) [inline app/templates/ember_widgets/documentation.hbs]: Thanks, guys. Phil asked over Slack, and it looks like he's on it.

**Comment by philpee2** [comment]
bump

File: app/templates/ember_widgets/documentation.hbs [hunk 2]
```diff
@@ -163,7 +169,7 @@
     </tr>
     <tr>
       <td>fade</td>
-      <td>yes</td>
+      <td>true</td>
       <td><p>If this is yes then the modal fades in/out.</p></td>
```
Comment (NicholasMohr) [inline app/templates/ember_widgets/documentation.hbs]: can you change all of these to `if this is true` if you're going to use true/false

Comment (philpee2) [inline app/templates/ember_widgets/documentation.hbs]: Good catch. Fixed

**Comment by NicholasMohr** [comment]
why do we have so many views/components in one file, isn't that against convention?

File: src/multi_select.coffee [hunk 3]
```diff
@@ -15,6 +15,11 @@ Ember.Widgets.MultiSelectOptionView = Ember.View.extend
     @notifyPropertyChange 'label'
   , 'context', 'labelPath'
 
+Ember.Widgets.MultiSelectTooltipItemView = Ember.Widgets.SelectTooltipOptionView.extend
+  processDropDownShown: ->
+    @_super()
+    @get('controller').focusTextField()
```
Comment (NicholasMohr) [inline src/multi_select.coffee]: why do we not need something like this in `SelectTooltip`?

**Comment by philpee2** [comment]
@NicholasMohr Generally yes, but it's done all over the place in Ember Widgets

File: src/multi_select.coffee (hunk 3 above)
Comment (philpee2) [inline src/multi_select.coffee]: Not sure, but that's not a new change in this commit. This is just adding the same from the regular view to the tooltip view

**Comment by NicholasMohr** [comment]
LGTM then

**Pull request merged**